topics/templates/document-attributes-product.adoc

```diff
@@ -11,6 +12,10 @@ ifeval::[{project_product_cd}==false]
 :project_name_full: Red Hat Single Sign-On
 :project_version: 7.3.0.GA
 :project_versionDoc: 7.3
+:project_previous_version: 7.2
+:project_previous_templates_version: sso-cd
```

Comment:
The _project_previous_templates_version_ is again used in `topics/tutorials.adoc` 30-ty times this time:
```
[iankko@localhost openshift]$ grep -rHn 'project_previous_templates_version' * | wc -l
30
```

I would like to keep this one, if possible a bit (especially since the previous templates version is different across the products, and it will need to be changed again once 7.3 is GA).

Reply:
Dropped `project_previous_templates_version` and refactored _Manual & Automatic DB Migration_ sections not to actually need it, see https://github.com/keycloak/keycloak-documentation/pull/496/commits/98cc71a8ba955182ff1629e1bc4719efe35597e8 for details.